packages/postgres-database/src/simcore_postgres_database/models/services.py

```diff
@@ -77,13 +98,25 @@
             onupdate="CASCADE",
             ondelete="CASCADE",
         ),
+        doc="Group Identifier",
     ),
+    # Access Rights flags ---
+    #  - Notice that read_access Flag is implicit on the existence of the row
```

Comment:
well there is no read access. that is why it is not here. if you have no execute/write access then you do not see the service.

Reply:
that is indeed what I meant by "implicit"

Reply:
yes but I don't understand why read then? what means reading a service?

Reply:
ok, now i understand your point. yes, read does not have a meaning for services. true